packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/model/FlutterSerializedModel.kt

```diff
@@ -48,7 +48,7 @@ data class FlutterSerializedModel(val serializedModel: SerializedModel) {
         return serializedData.mapValues {
             when (val value: Any = it.value) {
                 is Temporal.DateTime -> value.format()
-                is Model? -> FlutterSerializedModel(value as SerializedModel).toMap()
+                is Model -> FlutterSerializedModel(value as SerializedModel).toMap()
```

Comment:
I think this might solve a problem I was having.

Reply:
haha yeah, if there is a null for any field in your model, you are doomed.